Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -108,7 +108,7 @@ Object.defineProperties(Instanced3DModel3DTileContent.prototype, {
 
```

Comment:
Not included in this diff: Hm, I should also think about `Implicit3DTileContent`, should that report the size of the subtree/content metadata?

Reply:
Hm, this could be a small follow-up PR since this PR is already pretty large.

Reply:
I talked with @lilleyse about this, we decided to keep the statistics at the model level for now.